### ISSUE TITLE
re-find must match against a string

### ISSUE THREAD

**Issue by kalbert312** [rogeriochaves/npm-force-resolutions#29]
Suddenly I'm getting the error
```/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/cljs/core.cljs:9987
    (throw (js/TypeError. "re-find must match against a string."))))
           ^
TypeError: re-find must match against a string.
    at cljs$core$re_find (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/cljs/core.cljs:9987:12)
    at switch__2144__auto__ (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:39:3)
    at /Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:39:3
    at npm_force_resolutions$core$fetch_resolved_resolution_$_state_machine__2145__auto____1 (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.js:143:4)
    at cljs.core.async.impl.ioc-helpers/FN-IDX (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.js:159:94)
    at cljs.core.async.impl.ioc-helpers/run-state-machine (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/cljs/core/async/impl/ioc_helpers.cljs:35:23)
    at cljs.core.async.impl.ioc-helpers/run-state-machine-wrapped (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/cljs/core/async/impl/ioc_helpers.cljs:39:6)
    at /Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:39:3
    at Immediate.cljs$core$async$impl$dispatch$process_messages (/Users/kyle/.npm/_npx/99143/lib/node_modules/npm-force-resolutions/out/cljs/core/async/impl/dispatch.cljs:18:7)
    at processImmediate (internal/timers.js:461:21)
```

My resolutions block:

```
  "resolutions": {
    "react": "~16.14.0",
    "react-dom": "~16.14.0"
  }
```

Any help would be appreciated.

**Comment by kalbert312** [comment]
Nevermind, I did not check closed issues. https://github.com/rogeriochaves/npm-force-resolutions/issues/28

**Comment by rogeriochaves** [comment]
@kalbert312 no worries, you should not have to check closed issues, this is indeed a new issue (or at least the previous one should be reopened), anyway it should be fixed now by d1b4b0eb4be68bc6a3a961e62a5f0e06331da74c, and released on `v0.0.10`, let me know if any problems
